### PR TITLE
docker: fix emsdk build stage

### LINKED ISSUE #222: docker build failing on install emscripten layer
To reproduce:
```
docker build .
```

```
Unpacking '/opt/emsdk/zips/c630da9163a64e08de3dd948be0a0f7a175d285b-wasm-binaries.tbz2' to '/opt/emsdk/upstream'
Done installing tool 'releases-upstream-c630da9163a64e08de3dd948be0a0f7a175d285b-64bit'.
Running post-install step: npm ci ...
Running post-install step: npm install google-closure-compiler-linux
Done running: npm ci
Done installing SDK 'sdk-releases-upstream-c630da9163a64e08de3dd948be0a0f7a175d285b-64bit'.
Setting the following tools as active:
   node-12.18.1-64bit
   releases-upstream-c630da9163a64e08de3dd948be0a0f7a175d285b-64bit

Next steps:
- To conveniently access emsdk tools from the command line,
  consider adding the following directories to your PATH:
    /opt/emsdk
    /opt/emsdk/node/12.18.1_64bit/bin
    /opt/emsdk/upstream/emscripten
- This can be done for the current shell by running:
    source "/opt/emsdk/emsdk_env.sh"
- Configure emsdk in your bash profile by running:
    echo 'source "/opt/emsdk/emsdk_env.sh"' >> $HOME/.bash_profile
hack emscript config getter (em-config)
cp: cannot stat '/root/.emscripten': No such file or directory
The command '/bin/sh -c EMSDK_VERSION=1.39.2 && 	EMSDK_HASH=04d13b99d0630496139fa8f424c62aad65588c7b && 	mkdir -p /opt/emsdk && 	curl -sSL https://github.com/emscripten-core/emsdk/archive/${EMSDK_HASH}.tar.gz | tar zxf - -C /opt/emsdk --strip-components=1 && 	cd /opt/emsdk && 	./emsdk update && 	./emsdk install ${EMSDK_VERSION} && 	./emsdk activate ${EMSDK_VERSION} && 	echo "hack emscript config getter (em-config)" && 	cp /root/.emscripten /home/${APP_USER}/.emscripten && 	printf '#!/usr/bin/env python\nimport os, sys\nexecfile(os.getenv("HOME")+"/.emscripten")\nprint eval(sys.argv[1])\n' >/usr/local/bin/em-config && 	chmod a+x /usr/local/bin/em-config' returned a non-zero code: 1
```

### PR DESCRIPTION
**Pull request by tmatth**
Fixes #222

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -132,16 +132,16 @@ RUN \
...
 # install emscripten
 RUN \
-	EMSDK_VERSION=1.39.2 && \
-	EMSDK_HASH=04d13b99d0630496139fa8f424c62aad65588c7b && \
+	EMSDK_VERSION=1.40.1 && \
+	EMSDK_HASH=e88a3c5bbfef172a5b947768204ef734e2fb6e04 && \
 	mkdir -p /opt/emsdk && \
 	curl -sSL https://github.com/emscripten-core/emsdk/archive/${EMSDK_HASH}.tar.gz | tar zxf - -C /opt/emsdk --strip-components=1 && \
 	cd /opt/emsdk && \
 	./emsdk update && \
```
Comment (tdaede) [inline Dockerfile:140]: when I was messing with this, it seems that the "emsdk update" actually updates us to the latest version, making the EMSDK_VERSION sort of irrelevant. Maybe we should drop it?

Comment (tmatth) [reply Dockerfile:140]: I'd go the other route and drop the `emsdk update` (if possible).

Comment (tmatth) [reply Dockerfile:140]: Actually based on https://emscripten.org/docs/getting_started/downloads.html#updating-the-sdk I think `emsdk update` just fetches the latest list but you can still use an older version like we do here.

Comment (tdaede) [reply Dockerfile:140]: I think the problem is that the subsequent emsdk install then uses the newer version in the list if we don't explicitly provide it. I see a "master" zip from Jan 4 being downloaded and installed in the logs.

Comment (tmatth) [reply Dockerfile:140]: but we do explicitly provide it? it's `emsdk install ${EMSDK_VERSION}`

Comment (tdaede) [reply Dockerfile:140]: hmm ok. I guess I'm still confused why the docker build broke because of this but that seems right.

**Review by tdaede — APPROVED** [review]

**Pull request merged**